components/prize_pool/commons/prize_pool_legacy.lua

```diff
@@ -170,17 +170,16 @@ function LegacyPrizePool.mapSlot(slot, mergeSlots)
 
 	local opponentsInSlot = #slot
 	Table.iter.forEachPair(CACHED_DATA.inputToId, function(parameter, newParameter)
-		local input = slot[parameter]
-		if newParameter == 'seed' then
-			LegacyPrizePool.handleSeed(newData, input, opponentsInSlot)
+		local input = slot[parameter] ~= 'q' and slot[parameter] or CHECKMARK ---@type string?
 
-		elseif input and tonumber(input) ~= 0 then
-			-- Handle the legacy checkmarks, they were set in value = 'q'
-			-- If want, in the future this could be parsed as a Qualification instead of a freetext as now
-			if input == 'q' then
-				input = slot.link and Page.makeInternalLink(CHECKMARK, slot.link) or CHECKMARK
+		if newParameter == 'seed' then
```

Comment:
there is no 'seed' in case of #2319, so it would never be handled

Reply:
Same change as 
https://github.com/Liquipedia/Lua-Modules/pull/2319/files#diff-a34f2a6370da59733def09190ea82e4928e01f78aa26c944c39726dbb56633a2R294

Reply:
That should work, yes